ext/openssl/ossl.h

```diff
@@ -35,6 +35,13 @@
 #if !defined(OPENSSL_NO_OCSP)
 #  include <openssl/ocsp.h>
 #endif
+#if !defined(LIBRESSL_VERSION_NUMBER) && (OPENSSL_VERSION_NUMBER >= 0x10100000L)
+#  include <openssl/bn.h>
+#  include <openssl/rsa.h>
+#  include <openssl/dsa.h>
+#  include <openssl/evp.h>
+#  include <openssl/dh.h>
+#endif
```

Comment:
They can be unconditionally included since they all exist in OpenSSL 1.0.1 (which is the minimum supported) and LibreSSL.